src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,17 @@
+package calculator
+
+class StringAddCalculator(
+    private val translator: Translator
+) {
+
+    fun add(text: String?): Int {
+        if (text.isNullOrBlank()) return DEFAULT_RESULT_VALUE
+
+        return translator.run(text)
+            .sumOf { it.value }
+    }
+
+    companion object {
+        const val DEFAULT_RESULT_VALUE = 0
```

Comment:
```suggestion
        private const val DEFAULT_RESULT_VALUE = 0
```
클래스 내부에서만 접근한다면, private 접근자를 붙여보면 어떨까요?